### ISSUE TITLE
Padding side in Qwem-Embedding

### ISSUE THREAD

**Issue by khalil-Hennara** [QwenLM/Qwen3-Embedding#65]
Hi,
First thanks for your valuable contribution with Qwen-Embedding models, I've try to fine-tune the model using Sentence Transformers framework, but I've notice that when I've used padding_side='left' the loss start with 14 and this almost as if I am training with random initialization, So I've try to set padding_side='right', and it's give me an initial loss with 3 So, is there an error with the model card, or I've missed something.
